model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -181,11 +192,28 @@ def make_model_config(param_combo, model, model_repository,
 
         return model_config
 
-    @staticmethod
-    def get_base_model_config_dict(model_repository, model_name):
-        config = ModelConfig.create_from_file(
-            f'{model_repository}/{model_name}')
-        return config.get_config()
+    @classmethod
+    def get_base_model_config_dict(cls, config, client, gpus, model_repository,
+                                   model_name):
+        """ 
+        Attempts to create a base model config dict from config.pbtxt, if one exists
```

Comment:
This comment should go to wherever the actual code went.

Reply:
Agreed. I've left it here and also replicated it for create_model_config_dict